src/lib/components/events/EventCardGrid.svelte

```diff
@@ -1,17 +1,29 @@
 <script lang="ts">
-	import { Button } from '@emerald-dao/component-library';
 	import Icon from '@iconify/svelte';
 	import type { Event } from '$lib/types/event/event.interface';
+	import { compareDates } from '$lib/utilities/compareDates';
```

Comment:
Another suggestion here..
Why don't we make a folder inside utilities called `/dates`.
That way we store al utility functions related to dates in the same directory..